test/JDBC/upgrade/17_5/schedule

```diff
@@ -613,9 +613,10 @@ sys-login-property
 sys-fn-varbintohexsubstring
 BABEL_OBJECT_RESOLUTION_IN_ROUTINES
 alter-mvu-test
-test_constraint_like
+test_constraint_like-17-5-or-16-9
```

Comment:
this is bit unconventional. We should rename existing test to before_17_5_or_16_9 instead of doing this way

Reply:
As discussed offline, this is because only one version has specific output